### PR TITLE
docs(operators): add documentation for skipWhile

### LINKED ISSUE #99: Add docs for operator : filtering - skipWhile
Add docs for operator skipWhile under group filtering 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by ashwin-sureshkumar**
Close #99

### CODE REVIEW

File: src/operator-docs/filtering/skipWhile.ts [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+      code: `
+      import { range } from 'rxjs/observable/range';
+      import { skipWhile } from 'rxjs/operators';
+
+      const source = range(1, 20);
+      // skips all values until the first divisible by 7, then emit all values from then on
+      const example = source.pipe(
+                              skipWhile(val => val % 7 !== 0)
+                            );
+      const subscribe = example.subscribe(val => console.log(val));
```
Comment (sumitarora) [inline src/operator-docs/filtering/skipWhile.ts]: Could you add expected output as comments?

File: src/operator-docs/filtering/skipWhile.ts [hunk 2]
```diff
@@ -1,6 +1,44 @@
 import { OperatorDoc } from '../operator.model';
 
 export const skipWhile: OperatorDoc = {
-  'name': 'skipWhile',
-  'operatorType': 'filtering'
+  name: 'skipWhile',
+  operatorType: 'filtering',
+  signature: `
+    public skipWhile(predicate: Function): Observable<T>
```
Comment (JWO719) [inline src/operator-docs/filtering/skipWhile.ts]: Do we stick to Function as signature type or should we use arrow function to provide the return type? Example for this on could be: `public skipWhile(predicate: () => boolean): Observable<T>`

Comment (btroncone) [reply src/operator-docs/filtering/skipWhile.ts]: I think we also decided not to include generics in signatures, so we can remove `<T>`.

**Comment by ashwin-sureshkumar** [comment]
@JWO719 @sumitarora done, please re-review

**Comment by ladyleet** [comment]
Thx @sumitarora! And @ashwin-sureshkumar

**Pull request merged**